src/backend/TrafficCourtsApi/DisputeApi.Web/Features/TicketService/Models/Ticket.cs

```diff
@@ -6,10 +6,25 @@ public class Ticket
     {
         [Key]
         [Required]
-        public int TicketNumber { get; set; }
-        public string Name { get; set; }
-        public string DateOfIssue { get; set; }
-        public string TimeOfIssue { get; set; }
-        public string DriversLicence { get; set; }
+        public int Id { get; set; }
+        public string UserId { get; set; }
+        public string ViolationTicketNumber { get; set; }
+        public string CourtLocation { get; set; }
+        public string ViolationDate { get; set; }
+        public string SurName { get; set; }
+        public string GivenNames { get; set; }
+        public string Mailing { get; set; }
+        public string Postal { get; set; }
+        public string City { get; set; }
+        public string Province { get; set; }
+        public string Licence { get; set; }
+        public string ProvLicense { get; set; }
+        public string HomePhone { get; set; }
+        public string WorkPhone { get; set; }
+        public string Birthdate { get; set; }
+        public bool LawyerPresent { get; set; }
+        public bool InterpreterRequired { get; set; }
+        public string InterpreterLanguage { get; set; }
+        public bool CallWitness { get; set; }
```

Comment:
This is more of dispute fields than a ticket field. I am assuming dispute will have it's own entity and because it's possible to have more than one dispute on a ticket, we should look at separate dispute information from ticket information.

Reply:
I will remove these from the Ticket model for now